### PR TITLE
Optimistic block processing RFC

### PR DESCRIPTION
**Pull request by codchen**
- adds template for RFC in Sei
- adds RFC for optimistic block processing

### CODE REVIEW

File: docs/rfc/README.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
+---
+order: 1
+parent:
+  order: false
+---
+
+# Requests for Comments
+
+A Request for Comments (RFC) is a record of discussion on an open-ended topic
+related to the design and implementation of Tendermint Core, for which no
```
Comment (philipsu522) [inline docs/rfc/README.md]: nit - doesn't have to just be Tendermint

File: docs/rfc/rfc-000-optimistic-proposal-processing.md [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+
+The OP goroutine would operate on top of a cache branch of the Cosmos store, and a branch
+equivalent for any state that is not managed by the Cosmos store.
+
+The OP goroutine would periodically (e.g. after every 10 txs) check if a termination signal is sent
+to it, and stops if so. If not, the OP goroutine would set the completion signal when it finishes
+processing.
+
+Upon receiving a `ProcessProposal` call, the application would adopt the following procedure:
+> if round == 0
```
Comment (philipsu522) [inline docs/rfc/rfc-000-optimistic-proposal-processing.md]: might be good to call out why we only do OP for round 0 and not subsequent rounds

**Review by philipsu522 — APPROVED** [review]

File: docs/rfc/README.md (hunk 1 above)
Comment (codchen) [reply docs/rfc/README.md]: nice catch

File: docs/rfc/rfc-000-optimistic-proposal-processing.md (hunk 2 above)
Comment (codchen) [reply docs/rfc/rfc-000-optimistic-proposal-processing.md]: :+1:

**Pull request merged**